bundles/framework/hierarchical-layerlist/Flyout.js

```diff
@@ -162,15 +162,16 @@ Oskari.clazz.define('Oskari.framework.bundle.hierarchical-layerlist.Flyout',
          * @private
          */
         _getLayerGroups: function (groupingMethod) {
-            var me = this,
-                groupList = [];
+            var me = this;
+            var groupList = [];
 
             var allGroups = (me._currentFilter) ? me.mapLayerService.getFilteredLayerGroups(me._currentFilter) : me.mapLayerService.getAllLayerGroups();
 
             var layers = (me._currentFilter) ? me.mapLayerService.getFilteredLayers(me._currentFilter) : me.mapLayerService.getAllLayers();
             var layersCopy = layers.slice(0);
             var notLoadedBackend = {};
 
+            // TODO: is this needed anymore? Now that groups are coming with the layers...
```

Comment:
It would be preferable to answer this question and update the code instead of adding a TODO.